bundles/org.openhab.core.automation.module.script/src/main/resources/OH-INF/config/script-profile.xml

```diff
@@ -7,14 +7,27 @@
 	<config-description uri="profile:transform:SCRIPT">
 		<parameter name="toItemScript" type="text">
 			<label>Thing To Item Transformation</label>
-			<description>The Script for transforming state updates and commands from the Thing handler to the item. The script
+			<description>The Script for transforming state updates and commands from the Thing handler to the Item. The script
```

Comment:
Why did you change from "item" to "Item"? IMO "item" is correct and "Script/Thing" should be changed to "script/thing" in that sentence.

Reply:
I thought that openHAB terms like Item or Thing should be written uppercase

Reply:
How to continue here?

Reply:
I have written `item` lowercase and have not changed Script/Thing yet.
I would suggest to leave it as is and probably discuss a convention across all maintainers so no one is confused anymore whether Item has to be spelled upper- or lowercase.